src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -153,7 +158,11 @@ public void Initialize()
                 {
                     if (!_webHostSettings.IsSelfHost)
                     {
+#if MONO
+                        Task.Run(() => RunAndBlock());
```

Comment:
Maybe not super important for what it's intended, but this will behave differently since we do signal the cancellation token that is provided to the function (if bound) to allow the function to gracefully cleanup when the runtime signals a shutdown. 
I guess we could register to be notified of shutdown (assuming that's also available) and pass our own token that we signal ourselves when notified, but again, probably not worth it for now.
A comment here might be helpful, though. Thoughts?